### DISCUSSION TITLE
[Bug]: Tag description incorrect

### DISCUSSION THREAD

**Discussion by kathrinyna11** [GetPublii/Publii#1349] [Bug Report] [answered]
### Operating system

Mac OS Ventura

### Publii version

 0.44.4

### Post editor

None

### Bug description

Hi there,

We have a problem with the tag description.
I duplicated our website to translate it into english and now the source code shows in the tag description part.
https://hk-vr.de/applications/
It still works on the original website - all settings are similar.
https://hk-vr.de/deutsch/anwendungen/

We use a custom theme on Publii called HK (v1.2.0.0)

Any ideas?

Thanks in advance and best regards,

Kathrin

**Comment by bobmitro** [comment, marked as answer]
All you have to do is edit the tag and author .hbs files ‎⁨(in the following location: Documents ▸ ⁨Publii⁩ ▸ ⁨sites⁩ ▸ ⁨YOUR_SITE ▸ ⁨input⁩ ▸ ⁨themes⁩ ▸ ⁨YOUR_THEME) and change this code: {{description}} to {{{description}}}, adding one more set of bracket. 
This change was announced with the Publii 0.42 release: https://getpublii.com/blog/release-042.html

**Reply by kathrinyna11** [reply]
Thank you Bob! First time in a while that I am working on the website again :)
